### PR TITLE
Add autoconsent test page

### PR DESCRIPTION
**Pull request by sammacbeth**
This is a basic test page for consent popup management.

### CODE REVIEW

File: features/autoconsent/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Autoconsent Test</title>
+
+  <style>
+      /* styles, if needed */
+  </style>
```
Comment (kdzwinel) [inline features/autoconsent/index.html:10]:
```suggestion
```

🧼

File: features/autoconsent/main.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+// populate test sites
+const testSites = {
```
Comment (kdzwinel) [inline features/autoconsent/main.js:2]: why this format is easier than keeping it in HTML? We use this format elsewhere and it's easier to update in the future?

**Review by kdzwinel — APPROVED** [review]
Thanks @sammacbeth ! Looks great, but please link this test page from index.html + I left one more comment. Feel free to merge after that.

File: features/autoconsent/main.js (hunk 2 above)
Comment (sammacbeth) [reply features/autoconsent/main.js:2]: This way is less verbose, and allows test-cases to be easily copied over to/from the tests in the autoconsent project: https://github.com/cliqz-oss/autoconsent/blob/master/addon/test.ts#L82-L91

**Pull request merged**